src/main/java/com/yelp/nrtsearch/server/luceneserver/field/DateTimeFieldDef.java

```diff
@@ -243,6 +268,20 @@ private long getTimeToIndex(String dateString) {
     return date.getTime();
   }
 
+  private long getTimeFromEpochMillisString(String epochMillisString) {
+    String format =
+        String.format(
+            "%s could not parse %s as date_time with format %s",
+            getName(), epochMillisString, dateTimeFormat);
+    try {
+      long epochMillisLong = Long.parseLong(epochMillisString);
+      Instant epochMillisInstant = Instant.ofEpochMilli(epochMillisLong);
```

Comment:
Instead of converting to an Instant we can probably just validate this by checking that epochMillisLong is positive. If epochMillisString is parsed to a long without an error I think the number being negative is the only invalid state possible

Reply:
What if it is a time before the epoch, wouldn't that be negative?

Reply:
@aprudhomme good point. Then we don't need any additional validation at all

Reply:
Ok, in this case the only validation is parsing input as to `long`